Observer/AdyenBoletoDataAssignObserver.php

```diff
@@ -66,6 +67,9 @@ public function execute(Observer $observer)
         // Remove remaining brand_code information from the previous payment
         $paymentInfo->unsAdditionalInformation('brand_code');
 
+        // Remove cc_type information from the previous payment
+        $paymentInfo->unsAdditionalInformation('cc_type');
```

Comment:
`cc_type` is not an additional information. It's field in the BD table. It can't be unset with the given command.